### ISSUE TITLE
Unable to start PaaS in rootless docker [with proposed solution]

### ISSUE THREAD

**Issue by rpgoldman** [AI-Planning/planning-as-a-service#61]
I *believe* that this is due to my having rootless docker, and not being able to `make` the `server` as root:

```
Error response from daemon: driver failed programming external connectivity on endpoint server_frontend_1 (8097ffb8ff258274cb621ad256dc8f6930d1c09f047d848cf53456fdf42dc265): Error starting userland proxy: error while calling PortManager.AddPort(): reply.Error: map[desc:bad request: add_hostfwd: slirp_add_hostfwd failed]
```

It looks to me like the frontend server would like to run on port 80.

So I changed the ports to `8010:80`, so I wouldn't be taking over the host machine's port 80 and now it starts.

I'm afraid I don't know docker compose syntax well enough to make an MR myself, but I think what  would be best would be to have the docker compose file check to see if it's run as root and if not, open a different port, possibly one specified to the makefile like:

    make WEBPORT=8010

or something along those lines.

**Comment by haz** [comment]
Aye, even with root, it would fail. Port clashes can be frustrating :P. I think we should be able to have this included in the ENV stuff -- already including lots there: https://github.com/AI-Planning/planning-as-a-service/blob/main/server/.env.example

**Comment by haz** [comment]
Should be fixed with #62
